libs/components/forms/src/lib/modules/file-attachment/shared/file-size.pipe.ts

```diff
@@ -1,4 +1,4 @@
-import { DecimalPipe } from '@angular/common';
+import { formatNumber } from '@angular/common';
```

Comment:
We were injecting `DecimalPipe` which was causing a few problems with undefined injectors with the new standalone components. We're only using `DecimalPipe` to format the number; Angular exports the same utility function `formatNumber` in its public API, so we can forego injecting `DecimalPipe` altogether.